### ISSUE TITLE
grpc-js: Upgrade to grpc-js@1.8.19 causes TypeError: Cannot read properties of undefined (reading 'id')

### ISSUE THREAD

**Issue by meeham04** [grpc/grpc-node#2520]
### Problem description
Upon upgrading the `grpc-js` package from version `1.8.18` to `1.8.19`, our existing code that utilizes `getFoos` (a getter for a domain object implementing `grpc.ClientUnaryCall`) method fails with the error: `TypeError: Cannot read properties of undefined (reading 'id')`. This error seems to originate from the `transport.js` file. Everything works as expected with the `1.8.18` version.

### Reproduction steps
1. Start with a project using the `grpc-js` package version `1.8.18`.
2. Upgrade the `grpc-js` package to version `1.8.19`.
3. Call the `getFoos` method with appropriate input.
4. Observe the error that originates from `transport.js`.

Unfortunately, I am unable to provide a repository due to it being privately hosted. However, I can assure you that the only change is the upgrade of the `grpc-js` package version.

### Environment
 - OS name, version and architecture: Mac OS 13.4.1, M1
 - Node version: v18.16.0
 - Node installation method: nvm
 - Package name and version: `grpc/grpc-js@1.8.19`

### Additional context
Here's the full error stack:

```
server/node_modules/@grpc/grpc-js/build/src/transport.js:194
logging.trace(constants_1.LogVerbosity.DEBUG, 'keepalive', '(' + this.channelzRef.id + ') ' + this.subchannelAddressString + ' ' + text);
^

TypeError: Cannot read properties of undefined (reading 'id')
at Http2Transport.keepaliveTrace (/Users/foo/my-project/packages/server/node_modules/@grpc/grpc-js/build/src/transport.js:194:91)
at Http2Transport.maybeStartKeepalivePingTimer (/Users/foo/my-project/packages/server/node_modules/@grpc/grpc-js/build/src/transport.js:298:18)
at new Http2Transport (/Users/foo/my-project/packages/server/node_modules/@grpc/grpc-js/build/src/transport.js:104:18)
at ClientHttp2Session.<anonymous> (/Users/foo/my-project/packages/server/node_modules/@grpc/grpc-js/build/src/transport.js:537:25)
at Object.onceWrapper (node:events:628:26)
at ClientHttp2Session.emit (node:events:513:28)
at emit (node:internal/http2/core:330:3)
at process.processTicksAndRejections (node:internal/process/task_queues:84:21)
```

I hope this provides enough information to address the issue. Please let me know if you need any more details.

**Comment by murgatroid99** [comment]
This is fixed in 1.8.20.

**Comment by edwardrf** [comment]
For anyone end up here due to issues with gRPC, make sure your grpc-js version matches the version used by pulumi.
